drivers/counter/Kconfig.stm32_rtc

```diff
@@ -81,4 +81,11 @@ config COUNTER_RTC_STM32_BACKUP_DOMAIN_RESET
 	help
 	  Force a backup domain reset on startup
 
+config COUNTER_RTC_STM32_SAVE_VALUE_BETWEEN_RESETS
```

Comment:
Seems like this option is not used in the patch.

Reply:
Thanks! I fixed it